config/rbac/topolvm_node_role.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: topolvm-node
+rules:
+- apiGroups:
+    - ""
+  resources:
+    - nodes
+  verbs:
+    - get
+    - list
+    - watch
+    - update
+    - patch
+- apiGroups:
+    - topolvm.cybozu.com
+  resources:
+    - logicalvolumes
+    - logicalvolumes/status
+  verbs:
+    - get
+    - list
+    - watch
+    - create
+    - update
+    - delete
+    - patch
+- apiGroups:
+    - storage.k8s.io
+  resources:
+    - csidrivers
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - security.openshift.io
+  resources:
+    - securitycontextconstraints
+  verbs:
+    - use
+  resourceNames:
+    - privileged
+    #- topolvm-node TODO: this scc (topolvm-node) does not provide all the rights needed .. why?
```

Comment:
- i take it that `topolvm-node` isn't working and so you replaced with `privileged`?

Reply:
yes that is